src/actions/tags.js

```diff
@@ -51,7 +51,7 @@ export function createTagInCommunity (params, slug) {
   return {
     type: CREATE_TAG_IN_COMMUNITY,
     payload: {api: true, params, path: `/noo/community/${slug}/tag/`, method: 'POST'},
-    meta: {slug, optimistic: true}
+    meta: {slug, tag: params, optimistic: true}
```

Comment:
createTagInCommunity is only called when creating a tag in the tag moderation page.
the Create Tag button in post editor just updates the tagDescriptions locally until the post is saved